esp/tools/soapplus/httpproxy.cpp

```diff
@@ -674,25 +672,28 @@ class CSocksProxyThread : public Thread
                 return -1;
             }
 
```

Comment:
What are the first 8 bytes used for ?

Reply:
ok, I see below it is the port, correct ?

Reply:
Just a comment, perhaps it would be more clear if the port code was right after this read.

Reply:
To store the first 8 bytes, which are the header, includes version, command, destination ip and destination port.